src/webid-oidc.js

```diff
@@ -75,6 +75,8 @@ export async function logout(
         await fetch('/.well-known/solid/logout', { credentials: 'include' })
       } catch (e) {
         // Ignore errors for when we are not on a Solid pod
+        // But tell users it is harmless because they will see the GET failure in the console
+        console.err("Coulnd't find /.well-known/solid/logout, THIS IS HARMLESS.");
```

Comment:
Let's do `warn`, perhaps `info` even.

Also let's not use caps if it is harmless 🙂
Typo in first word.

Reply:
I did err() because the GET will be in err;  If we do info() and developer turns off all but errors, they's see the GET error but not the message.  Agreed on all caps. :-)

Reply:
I understand the reasoning, but given that this is an information message (not an error of itself), we probably want to not have it as prominent. So I am fine with it disappearing on filtering.

Reply:
Sounds good, thanks!